pkgs/applications/misc/apvlv/default.nix

```diff
@@ -43,6 +43,12 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  # poppler 0.73.0 moved functionality from gtypes.h into gfile.h
+  postPatch = ''
+    grep -rlF '#include "goo/gtypes.h"' | xargs sed -i 's|#include "goo/gtypes.h"|#include "goo/gfile.h"|g'
```

Comment:
The only one that matters here is https://github.com/naihe2010/apvlv/blob/41f68a9640dcf2c179567a7e8c8d3d0552ece584/src/ApvlvPdf.cc#L32

So a patch would be safer.

I conveniently have a patch lying around for this, if you'd like to include it
https://github.com/worldofpeace/nixpkgs/commit/1d34b851e608f69d1de154f599c7dbaab48327de

Reply:
Your solution looks more convenient indeed. I'll provide a new PR with your patch.